Library/Homebrew/extend/os/mac/dependency_collector.rb

```diff
@@ -0,0 +1,5 @@
+def ant_dep(spec, tags)
```

Comment:
Check out the other files in this directory to see the pattern for porting these.